### DISCUSSION TITLE
How to tell what version is running?

### DISCUSSION THREAD

**Discussion by DaddyW123** [LizardByte/community#74] [Sunshine] [answered]
I am attempting to set up sunshine on my windows PC.  I downloaded it a while back and I'm curious if the version that's here on Github is newer or the same as what I'm running.  But I can't seem to find any reference in my download as to what version it is.  I can't find it in the web ui.  And even though I'm currently running it as a service, if I run the sunshine.exe file to see it load, I don't see any references there to version number either.

**Comment by ReenigneArcher** [comment, marked as answer]
`sunshine --help` will show you list of command line args... one of which is `--version`

**Reply by DaddyW123** [reply]
Thank you for the quick response.  That was all I needed.  I am running 0.14.0 and the current version is 0.14.1.
